<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HM013</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>541</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>  功能</string>
     </attribute>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>320</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>460</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_3">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>320</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>460</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_4">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>460</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>400</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LeftMICFF        LeftMICFB       RightMICFF      RightMICFB</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>460</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_4">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>320</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>221</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>50</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>50</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_4">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>50</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_5">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>50</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string> 导入</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>             PGA</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_5">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_6">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_7">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_8">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> IR测试</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 触摸测试</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>     产 测 功 能</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>        音频场景</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 传感器测试</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>420</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> ANC PASS </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> ANC OFF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>380</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> ANC ON</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>日期：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>380</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>   查找USB</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;日志信息：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> 设置</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>中文</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> 语言(Language)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>400</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt; Designed:华 锦 文&lt;/p&gt;&lt;p&gt; Date:2020-6-20 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>261</width>
        <height>301</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;	&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;许可证&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:11pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:11pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:11pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; color:#ffffff;&quot;&gt;初次使用有效期是一个月，过期请联系&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;请输入Lincese&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>  压感</string>
     </attribute>
     <widget class="QCustomPlot" name="serial_plot" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>721</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>721</width>
        <height>111</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>   串口调试信息</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>   串口号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_7">
         <item>
          <property name="text">
           <string>无</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>    波特率</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>115000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1152000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自定义</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>    停止位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>    数据位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>  奇偶校验</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>    流控制</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_3">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_4">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_5">
         <item>
          <property name="text">
           <string>奇数</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>偶数</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_6">
         <item>
          <property name="text">
           <string>硬件</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>无</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>180</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string> 开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string> 搜寻</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>  发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>240</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>  清除</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string> 生产指标</string>
     </attribute>
     <widget class="QCustomPlot" name="graph_plot" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>691</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>1081</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Image/more.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
